library/src/scripts/flyouts/NewPostBackground.tsx

```diff
@@ -2,31 +2,30 @@ import React from "react";
 import classNames from "classnames";
 import { createPortal } from "react-dom";
 
-import { useSpring, animated } from "react-spring";
-import { newPostBackgroundClasses, newPostBackgroundVariables } from "./newPostBackgroundStyles";
-import { colorOut } from "@library/styles/styleHelpers";
+import { animated } from "react-spring";
+import { newPostBackgroundClasses } from "./newPostBackgroundStyles";
 
 interface IProps {
     open: boolean;
     className?: string;
     children: React.ReactNode;
-    onClick: () => void;
+    onClick: (e) => void;
+    onKeyDown?: (e) => void;
+    trans: any;
 }
 
 export default function NewPostBackground(props: IProps) {
     const classes = newPostBackgroundClasses();
-    const vars = newPostBackgroundVariables();
-
-    const trans = useSpring({
-        backgroundColor: props.open ? colorOut(vars.container.color.open) : colorOut(vars.container.color.close),
-        from: { backgroundColor: colorOut(vars.container.color.close) },
-        config: { duration: vars.container.duration },
-    });
 
     return createPortal(
-        <animated.div className={classNames(classes.container)} style={trans} onClick={props.onClick}>
+        <animated.aside
```

Comment:
Why is this an aside? It's not semantic to have click and keyboard events on that element. This is a button if you can click on it.

Reply:
Oops, this is the background, this doesn't make sense as a button, nevermind.

Reply:
I've moved the key handler to the button menu, so this portal is out of the flow of the page and it seems to me semantically like an aside now.